src/goto-programs/goto_program.cpp

```diff
@@ -126,6 +126,11 @@ std::ostream &goto_programt::output_instruction(
         out << "HAVOC_OBJECT " << format(code.op0()) << '\n';
         break;
       }
+      else if(code.get_statement() == ID_output)
+      {
+        out << "OUTPUT " << format(code.op0()) << '\n';
+        break;
+      }
```

Comment:
As much as I think `OTHER` should go away, for now we obviously have to cope with it. But then: should we at least try to make this more complete by also adding `ID_input`, `ID_fence`, `ID_printf`, `ID_array_copy`, `ID_array_replace`, ID_array_set? (And probably also `ASM`.)